ledger/accountdb.go

```diff
@@ -98,8 +98,13 @@ var creatablesMigration = []string{
 // createNormalizedOnlineBalanceIndex handles accountbase/catchpointbalances tables
 func createNormalizedOnlineBalanceIndex(idxname string, tablename string) string {
 	return fmt.Sprintf(`CREATE INDEX IF NOT EXISTS %s
-		ON %s ( normalizedonlinebalance, address, data )
-		WHERE normalizedonlinebalance>0`, idxname, tablename)
+		ON %s ( normalizedonlinebalance, address ) WHERE normalizedonlinebalance>0`, idxname, tablename)
```

Comment:
this removes the `data` column from the index but since this CREATE INDEX gets run in `performResourceTableMigration` if this was intended, won't this create a divergence from previously upgraded nodes? Or is this just a typo and we should bring back data to get it to be compatible with previously-upgraded nodes?

Reply:
```suggestion
		ON %s ( normalizedonlinebalance, address, data )
		WHERE normalizedonlinebalance>0`, idxname, tablename)
```

Reply:
I'll restore but in general I do not think data should be in the index

Reply:
We could kill it in another migration? Without preserving the old migration, we will end up with one class of nodes out there that still has it in the old index with data, and another class of nodes without data, and we won't know which is which.

Now that there is an online accounts table it doesn't look like we use this index for anything except creating the online accounts table during catchpoint restore.